databases.star

```diff
@@ -0,0 +1,118 @@
+POSTGRES_IMAGE = "postgres:16.2"
+POSTGRES_SERVICE_NAME = "postgres"
+POSTGRES_PORT = 5432
+POSTGRES_MASTER_DB = "master"
+POSTGRES_MASTER_USER = "master_user"
+POSTGRES_MASTER_PASSWORD = "master_password"
+
+NODE_DATABASES = {
+    "event_db": {
+        "name": "event_db",
+        "user": "event_user",
+        "password": "rJXJN6iUAczh4oz8HRKYbVM8yC7tPeZm",
+        "init": read_file(src="./templates/databases/event-db-init.sql"),
+    },
+    "pool_db": {
+        "name": "pool_db",
+        "user": "pool_user",
+        "password": "Qso5wMcLAN3oF7EfaawzgWKUUKWM3Vov",
+    },
+    "prover_db": {
+        "name": "prover_db",
+        "user": "prover_user",
+        "password": "SR5xq2KZPgvQkPDranCRhvkv6pnqfo77",
+        "init": read_file(src="./templates/databases/prover-db-init.sql"),
+    },
+    "state_db": {
+        "name": "state_db",
+        "user": "state_user",
+        "password": "rHTX7EpajF8zYDPatN32rH3B2pn89dmq",
+    },
+}
+
+DATABASES = NODE_DATABASES | {
```

Comment:
Can we find a better name, maybe `CDK_DATABASES`?

Reply:
Done!
https://github.com/0xPolygon/kurtosis-cdk/pull/125/commits/ac6609e92252cd788719129b0901d921ace00616